specs/WebResourceResponseReceived.md

```diff
@@ -1,31 +1,38 @@
 # Background
 The WebView2 team has been asked for an API to get the response for a web
 resource as it was received and to provide request headers not available when
-`WebResourceRequested` event fires (such as Authentication headers). The
-`WebResourceResponseReceived` event provides such HTTP response representation
-and exposes the request as sent over the wire.
+`WebResourceRequested` event is raised (such as HTTP Authentication headers).
+The `WebResourceResponseReceived` event provides such response representation
+and exposes the request as committed.
 
 In this document we describe the new API. We'd appreciate your feedback.
 
 # Description
-The `WebResourceResponseReceived` event fires when the WebView receives the
-response for a request for a web resource. It provides access to both the
-response as it was received and the request as it was sent over the wire,
-including modifications made by the network stack (such as adding of
-Authorization headers). The app can use this event to view the actual request
-and response for a web resource, but modifications made to these objects are
-ignored.
+The `WebResourceResponseReceived` event allows developers to inspect the
+response object from URL requests (such as HTTP/HTTPS, file and data). A key
+scenario is to allow developers to get Auth headers from an HTTP response to
+authenticate other tools they're using, since the Auth headers are not exposed
+in the `WebResourceRequested` event.
+
+This event is raised when the WebView receives the response for a request for a
+web resource. It provides access to both the response as it was received and the
+request as it was committed, including modifications made by the network stack
+(such as the adding of HTTP Authorization headers). The app can use this event
+to view the actual request and response for a web resource, but modifications
+made to these objects are ignored.
 
 The host app registers for this event by providing a
```

Comment:
It would be better if the docs here are agnostic of the API language / runtime. This sentence is specific to registering for events in our COM API. But I don't think we even need to describe how to do that. We have a separate doc on registering for events in our COM API. Maybe remove this first sentence, and start with the next `When the event is raised, the WebView will pass...`

Reply:
Thanks! I have updated.